gapic/api/artman_clouddebugger.yaml

```diff
@@ -11,7 +11,7 @@ common:
   gapic_api_yaml:
     - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml
   output_dir: ${REPOROOT}/artman/output
-  enable_batch_generation: True
+  enable_batch_generation: False
```

Comment:
I think delete it - having it be false suggests a reason. But don't have strong feelings, was more just checking whether there *was* a reason that I wasn't seeing.